java/arcs/core/data/proto/manifest.proto

```diff
@@ -289,3 +298,15 @@ message CheckProto {
   AccessPathProto access_path = 1;
   InformationFlowLabelProto.Predicate predicate = 2;
 }
+
+message PolicyConstraintsProto {
+  PolicyProto policy = 1;
+
+  InformationFlowLabelProto.Predicate egress_check = 2;
```

Comment:
Just doing a sanity check (no pun intended!). 

Did you think about this a bit more? It seem OK to just have a `Predicate` as the egress check when translating the policies, right?

Reply:
Yep, I think it makes sense!